converter/src/main/java/gov/cms/qpp/conversion/model/error/Detail.java

```diff
@@ -66,7 +66,7 @@ public static Detail forErrorCode(LocalizedError error) {
 		Objects.requireNonNull(error, "error");
 
 		Detail detail = new Detail();
-		detail.setErrorCode(error.getErrorCode());
+		detail.setErrorCode(error.getErrorCode().ordinal());
```

Comment:
I am not a fan of using the ordinal value.  :\  I outlined my reasoning in a previous comment.  https://github.com/CMSgov/qpp-conversion-tool/pull/417#discussion_r148096056

Reply:
Agreed.  My suggestion is to add `Integer errorCode` to `ErrorCode`, and then change the constructor to look something like: `ENCODER_MISSING(1001, "Failed to find an encoder")`.  Each error in the enum gets a specific, hardcoded number.  Or, it could be a string like "CT1001" or something.